runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -262,6 +263,10 @@ public static DataflowRunner fromOptions(PipelineOptions options) {
         "DataflowRunner requires stagingLocation, and it is missing in PipelineOptions.");
     validator.validateOutputFilePrefixSupported(dataflowOptions.getStagingLocation());
 
+    if (!Strings.isNullOrEmpty(dataflowOptions.getSaveProfilesToGcs())) {
```

Comment:
Not currently. We may add one in the future, but requiring that the user (and VMs) have access seems like the simplest path for now.